### ISSUE TITLE
Color / Contract Issue (Directory)

### ISSUE THREAD

**Issue by SteveRyan-ASU** [asuengineering/asu-labs-theme#15]
Within the display of the featured person on the directory page, there are details that are dark grey text on a black background. Need to change them to white text, regardless of if there is a link present within the `<li>` tag or not. 

Check template [archive-person.php](https://github.com/asuengineering/asu-labs-theme/blob/master/archive-person.php) and `_people.scss:238` for the offending rule.

## Additional Info
Color contrast is insufficient for room information text. Insufficient contrast between text and its background can give problems for users with visual impairments and color blindness, but it can also affect a lot of other users, leading to important text potentially being overlooked. Ensure that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text)

## Reference 
(Reference WCAG 2.1, 1.4.4 Contrast)
WCAG 2.1 Score: Level A
Potential User Barrier Level: Medium
